src/mixer/basetrackplayer.cpp

```diff
@@ -134,8 +136,41 @@ BaseTrackPlayerImpl::BaseTrackPlayerImpl(QObject* pParent,
             SLOT(slotWaveformZoomSetDefault(double)));
 
     m_pPreGain = std::make_unique<ControlProxy>(group, "pregain", this);
-    // BPM of the current song
 
+    m_pShiftCuesEarlier = std::make_unique<ControlPushButton>(
+            ConfigKey(group, "shift_cues_earlier"));
+    connect(m_pShiftCuesEarlier.get(),
+            &ControlObject::valueChanged,
+            this,
+            [this](double value) { slotShiftTimesButton(value, -1 * kShiftCuesOffsetMillis); });
```

Comment:
The slot names are not matching the co names now.

Reply:
oh, good catch

Reply:
done